frontend/public/components/dashboard/project-dashboard/project-dashboard.tsx

```diff
@@ -9,25 +9,50 @@ import { UtilizationCard } from './utilization-card';
 import { InventoryCard } from './inventory-card';
 import { ActivityCard } from './activity-card';
 import { ProjectDashboardContext } from './project-dashboard-context';
+import { LauncherCard } from './launcher-card';
+import { connect } from 'react-redux';
+import { getNamespaceDashboardConsoleLinks } from '../../overview/namespace-overview';
```

Comment:
I'm deleting this entire file in #3033. It would be better to move this helper to another location.

Reply:
That's OK, I can move it in #3033